smoke-tests/src/test/groovy/io/opentelemetry/smoketest/AppServerTest.groovy

```diff
@@ -270,7 +268,7 @@ abstract class AppServerTest extends SmokeTest {
     traces.countFilteredEventAttributes('exception.message', 'This is expected') == 1
 
     and: "The span for the initial web request"
-    traces.countFilteredAttributes("http.target", "/app/exception") == 1
+    traces.countFilteredAttributes("url.path", "/app/exception") == 1
```

Comment:
Can we use the constants here? like 
```suggestion
    traces.countFilteredAttributes(SemanticAttributes.URL_PATH.getKey(), "/app/exception") == 1
```

Reply:
Yeah, we actually can!
Will fix that, thanks 👍